README.md

```diff
@@ -29,14 +38,18 @@ The example uses the [World Cities Dataset](https://simplemaps.com/data/world-ci
   }
 ```
 
-### Database Setup
+## Database setup
 
 It's recommended to use [`Postgres`] for your database so the schema can use `BigInt` and `Float` nicely.
 
 Also, the seed script uses Prisma's [`createMany()`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany) to bulk load records in batches which is significantly faster that loading data row by row -- which is important because we have ~43,000 world cities in the dataset.
 
 Unfortunately, `createMany` is not supported by SQLite.
 
+### Setting up a database locally
+
+#### OSX
+
 The easiest way to get Postgres up ad running locally on OSX is [Postres.app](https://postgresapp.com).
 
 You can then set you database configuration settings to two different local databases, like:
```

Comment:
[question] Will most readers know what to do after this? Is there more they need to connect to the database in the app? Is there a tutorial/docs we could point them to? Right now, it kind of feels like a "draw the rest of the owl" situation, but I don't know much about Postgres.

Reply:
They shouldn't need to do more beyond applying the Prisma schema and running the seed script, but I noticed that those instructions are only mentioned in the 'railway' section.

I haven't pushed these changes yet but I'm creating another section for 'populating the data in the database after set up' which lists the commands